devices/namron.js

```diff
@@ -734,4 +734,29 @@ module.exports = [
         extend: extend.light_onoff_brightness_colortemp({colorTempRange: [153, 555]}),
         meta: {turnsOffAtBrightness1: true},
     },
+    {
+        zigbeeModel: ['4512749'],
+        model: '4512749',
+        vendor: 'Namron',
+        description: 'Namron - Zigbee Thermostat Outlet Socket',
+        fromZigbee: [fz.metering, fz.electrical_measurement, fz.on_off, fz.temperature],
+        toZigbee: [tz.on_off, tz.power_on_behavior],
+        exposes: [
+            e.temperature(), e.power(), e.current(), e.voltage(), e.switch(),
+            exposes.enum('power_on_behavior', ea.ALL, ['on', 'off', 'restore']).withDescription('Recover state after power outage'),
```

Comment:
`restore` is not accepted by `tz.power_on_behavior`, use `e.power_on_behavior()` instead here.